proto/lighthouse-result.proto

```diff
@@ -206,13 +206,25 @@ message LighthouseResult {
     // List of the audits that were skipped, empty if all were run
     // nullable list of strings
     google.protobuf.Value skip_audits = 15;
+
+    // Flag indidcating this audit was audit only or not
+    google.protobuf.BoolValue audit_mode = 16;
   }
 
   // The settings that were used to run this audit
   ConfigSettings config_settings = 12;
 
   // i18n info in version 1 message format
   I18n i18n = 13;
+
+  // Message containing timing information about the LHR
+  message Timing {
+    // The total time taken by the LHR
+     google.protobuf.DoubleValue total = 1;
```

Comment:
(timing PR isn't landed yet. i got an extension test blowing up because we nuked that try/catch :) 

but since it'll be landing soon.... 

> If we excluded it, it'd be the only property we're excluding in LR.

insert 👢 in 😮 

And 10 seconds later I'm all pro-excluding from the proto.. :)

Reply:
So...keep it in or?